### DISCUSSION TITLE
An unexpected error occurred while connecting Node

### DISCUSSION THREAD

**Discussion by Gabry2790** [EvieePy/Wavelink#271] [General]
An unexpected error occurred while connecting Node(identifier=..., uri=...
, status=NodeStatus.CONNECTING, players=0) to Lavalink: ".../v4/websocket"
If this error persists or wavelink is unable to reconnect, please see: https://github.com/PythonistaGuild/Wavelink/issues

**Comment by chillymosh** [comment]
Converting to discussion as it is not a lib issue.
@Gabry2790 You have offered no helpful information here. 
What version of Lavalink are you using? Are you hosting it locally, if so then what does the URI look like? 
What does your connect look like in setup_hook?

**Reply by Gabry2790** [reply]
Lavalink ver: 4.0.0
I tried hosting locally and using a lavalink server
URI is like `ip:port`

```py
async def setup_hook(self):
        node = [wavelink.Node(uri="", password="")]
        await wavelink.Pool.connect(nodes=node, client=self)
```

**Reply by chillymosh** [reply]
You are probably missing the protocol  `http://localhost:2333`  (If it's local it is helpful if you actually post this information for us)

**Reply by Gabry2790** [reply]
Now i'm not hosting it locally, but i tried that protocol and gave me the same error

**Reply by chillymosh** [reply]
If you are now trying to connect to someone else's public server then the may not be running v4.
If you are hosting this publicly then please provide the Lavalink startup logs. 
You can also join the Discord for more detailed assistance.

**Reply by Gabry2790** [reply]
I solved it, I had entered the wrong IP, sorry for wasting your time
